### DISCUSSION TITLE
Website doesn't work https://html2canvas.hertzen.com/

### DISCUSSION THREAD

**Discussion by singhhrpreet** [niklasvh/html2canvas#2672] [Q&A] [answered]
I recently wanted to use https://html2canvas.hertzen.com/ for a project. The website doesn't seem to work!

I see this 
![image](https://user-images.githubusercontent.com/20049511/130273364-4b6e647e-72f6-4222-b12e-c5830898c68f.png)

I tried various WiFi networks, office, home, mobile networks. I suspected this to be a geographical location-based restriction. 

Forwarded this link to a few of my friends, it doesn't work for them either.

Is anyone else facing this issue too?

**Comment by s-kris** [comment, marked as answer]
Can confirm!

**Reply by niklasvh** [reply]
Google analytics and https://www.isitdownrightnow.com/html2canvas.hertzen.com.html indicate otherwise - could your ISP be blocking certain hosts/ip's?

**Reply by s-kris** [reply]
Oh damn! Sorry about that. Should've checked on a VPN. It works.
